### PR TITLE
Experiment trials iteration cycle

### PR DESCRIPTION
**Pull request by msvinaykumar**
Experiment trials will get executed repeatedly as per iteration cycle count which derived from Analyser module.

Signed-off-by :msvinaykumar <vinakuma@redhat.com>

### CODE REVIEW

File: src/main/java/com/autotune/analyzer/layer/ContainerLayer.java [hunk 1]
```diff
@@ -39,18 +39,24 @@ public class ContainerLayer extends GenericLayer implements Layer {
     @Override
     public void prepTunable(Tunable tunable, JSONObject tunableJSON, ContainerConfigData containerConfigData) {
         String tunableName = tunable.getName();
-        Map<String, Quantity> requestPropertiesMap = new HashMap<String, Quantity>();
+
+        Map<String, Quantity> requestPropertiesMap =  containerConfigData.getRequestPropertiesMap();
+        if(null == requestPropertiesMap){
+            requestPropertiesMap = new HashMap<>();
+        }
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/layer/ContainerLayer.java:46]: Can you please fix the formatting / coding standards here

File: src/main/java/com/autotune/analyzer/layer/QuarkusLayer.java [hunk 2]
```diff
@@ -41,13 +41,18 @@ public void prepTunable(Tunable tunable, JSONObject tunableJSON, ContainerConfig
...
 		ArrayList<EnvVar> environmentalAmendList = (ArrayList<EnvVar>) containerConfigData.getEnvList();
+		String tunableName = tunable.getName();
+		if (tunableName.contains("quarkus")) {
+			tunableName = "-D" + tunableName;
+		}
 		if( environmentalAmendList.size() == 0) {
-			runtimeOptions.append(tunable.getName()).append(EQUALS).append(tunableJSON.get("tunable_value") ).append(" ");
+			runtimeOptions.append(tunableName).append(EQUALS).append(tunableJSON.get("tunable_value") ).append(" ");
 			environmentalAmendList.add(new EnvVar(JAVA_OPTIONS, runtimeOptions.toString(), null));
 		}else {
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/layer/QuarkusLayer.java:51]: Coding standards fix here as well

Comment (msvinaykumar) [reply src/main/java/com/autotune/analyzer/layer/QuarkusLayer.java]: done

File: src/main/java/com/autotune/common/k8sObjects/Metric.java [hunk 3]
```diff
@@ -115,14 +58,11 @@ public String getValueType() {
...
     }
 
-    @Override
-    public String toString() {
-        return "Metric{" +
-                "name='" + name + '\'' +
-                ", query='" + query + '\'' +
-                ", datasource='" + datasource + '\'' +
-                ", valueType='" + valueType + '\'' +
-                '}';
```
Comment (dinogun) [inline src/main/java/com/autotune/common/k8sObjects/Metric.java:125]: Is it intentional to remove toString here?

Comment (msvinaykumar) [reply src/main/java/com/autotune/common/k8sObjects/Metric.java]: No . I just removed thinking not required at this point in time . But now I added it back

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**